datasetinsights/data/datasets/synthetic.py

```diff
@@ -297,10 +307,57 @@ def exists(capture_file):
 
         return catalog[keep_mask]
 
-    def download(self, manifest_file):
+    def download_captures_from_manifest(self, manifest_file):
         """ Download captures of a given manifest file.
 
         Args:
             manifest_file (str): path to a manifest file
         """
         _download_captures(self.root, manifest_file)
+
+    @staticmethod
+    def download(source_uri, destination):
```

Comment:
Can you add a checksum for the public synthetic subset? You can use `from datasetinsights.data.download import validate_checksum`, which is well tested. Also, take a look at `_download_http` method in `groceries_real.py` as an example.

Reply:
@Saurav-D  Agree with @BlairLee. It's important (and typical) to provide checksums for large files. We should have simple checksum mechanism to make sure these files are not corrupted during the download process.

Reply:
@adason I've implemented the checksum, I will push the changes along with others. But I just wanted to know if we should make version of Synthdet dataset as describe in above comments.